test/distributed/checkpoint/test_state_dict.py

```diff
@@ -875,6 +875,55 @@ def test_setting_meta_device_model_broadcasting(self) -> None:
             )
             self.assertEqual(cpu_model_value, meta_model_value)
 
+    @with_comms
+    @skip_if_lt_x_gpu(1)
+    def test_memory_between_view_and_clone(self) -> None:
```

Comment:
Add memory check at `test_setting_meta_device_model_broadcasting` after calling `set_model_state_dict`
Test pass right now by the change if 'clone', but would fail if without the change in this PR